federation/develop/develop.sh

```diff
@@ -133,33 +127,6 @@ function push() {
 
   kube::log::status "Pushing hyperkube image to the registry"
   gcloud docker -- push "${KUBE_REGISTRY}/hyperkube-amd64:${kube_version}"
-
-  # Update config after build and push, but before turning up the clusters
-  # to ensure the config has the right image version tags.
-  gen_or_update_config "${kube_version}"
```

Comment:
Sorry why is this not required now?
It is deprecated but we dont want to intentionally break it :)